modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala

```diff
@@ -568,4 +568,14 @@ object SeqTranslate {
     def toAction(kind: ActionType): Action[F] = fromF[F](kind, x.map(r => Result.OK(Response.Configured(r.sys.resource))))
   }
 
+  def dataIdFromConfig[F[_]: MonadError[?[_], Throwable]](config: Config): F[DataId] =
```

Comment:
This is already defined at `ObserveActions`. see https://github.com/gemini-hlsw/ocs3/blob/develop/modules/seqexec/server/src/main/scala/seqexec/server/ObserveActions.scala#L110

Reply:
Not exactly. The one in `ObserveActions` takes a `ObserveEnvironment[F]`

Reply:
Could we merge them?

Reply:
Done